Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM openjdk:8-jre-alpine
+
+ARG VERSION
+
+LABEL com.blackducksoftware.integration.alert.vendor="Black Duck Software, Inc." \
```

Comment:
Should this be changed?

Reply:
I will double check with Ari.  Our stuff on docker hub all follows this convention.